MapboxSearch/base/src/main/java/com/mapbox/search/base/location/LocationEngineAdapter.kt

```diff
@@ -14,6 +14,18 @@ import com.mapbox.search.base.utils.LocalTimeProvider
 import com.mapbox.search.base.utils.TimeProvider
 import com.mapbox.search.base.utils.extension.toPoint
 import com.mapbox.search.internal.bindgen.LonLatBBox
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+// Maximum time period for waiting for location updates (in milliseconds)
+private var locationEngineObservationTimeout: Long? = 1000L
+
+@Suppress("UNUSED")
+fun setLocationEngineLocationObservationTimeout(timeout: Long?) {
```

Comment:
Why can't we just remove it?

Reply:
Someone may want to set a different timeout value but we don't call this function ourselves. That's why it's marked as `UNUSED`.

Reply:
> Someone may want to set a different timeout

[You aren't gonna need it](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it), so I think this can be removed for now. There's a chance you'll come up with a better solution later

Reply:
OK, I removed it.